kubernetes/charts-linera/templates/linera-server.yaml

```diff
@@ -1,4 +1,22 @@
 apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: linera-adminserviceaccount
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: linera-adminserviceaccount
+subjects:
+  - kind: ServiceAccount
+    name: linera-adminserviceaccount
+    namespace: default
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
+  apiGroup: rbac.authorization.k8s.io
```

Comment:
So will this also be used in "prod" validators? Asking because I've seen it being recommended against having admin level access service accounts in production, because of the risk of someone gaining access to it

Reply:
I added a comment in a subsequent PR. This is definitely not a good idea long-run, but for developing quickly it's going to be helpful to make everythihg admin and nail down permissions over tme.